thirdparty_builtin/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ if(ENABLE_TESTS)
         list(APPEND gtest_defines "-DGTEST_HAS_DEATH_TEST=0")
     endif()
 
+    if(gtest_disable_pthreads)
+      list(APPEND gtest_defines "-DGTEST_HAS_PTHREAD=0")
+    endif()
```

Comment:
It seems like if ``gtest_disable_pthreads`` is defined, then the following should already add this compiler define:
* https://github.com/google/googletest/blob/master/googletest/cmake/internal_utils.cmake#L58-L65
* https://github.com/google/googletest/blob/master/googletest/cmake/internal_utils.cmake#L121-L127

Is something else in your configuration undoing this?  (e.g. multiple blt projects, one of which is not setting ``gtest_disable_pthreads`` correctly?

Reply:
Yup, I saw that. I'm not sure why that doesn't work. I know it's not an issue with multiple blt projects though.